src/Domain/Creators/TimeDependentOptions/BinaryCompactObject.cpp

```diff
@@ -71,11 +72,91 @@ TimeDependentMapOptions<IsCylindrical>::TimeDependentMapOptions(
 }
 
 template <bool IsCylindrical>
+std::unordered_map<std::string,
+                   std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>
+TimeDependentMapOptions<IsCylindrical>::create_worldtube_functions_of_time()
+    const {
+  ASSERT(not translation_map_options_.has_value(),
+         "Translation map is not implemented for worldtube evolutions.");
```

Comment:
Make these `ERROR`s. If the user specifies an incorrect option, we should get an error and not be able to proceed